### PR TITLE
fix: add missing actions to policy.json

### PR DESCRIPTION
**Pull request by roblambell**
## Problem

Actions missing from the recommended [komiser policy](https://raw.githubusercontent.com/mlabouardy/komiser/master/policy.json)

## Solution

Add required actions to the policy (using https://aws.permissions.cloud/ as IAM <> API reference)

## Changes Made

- Updated policy.json

## How to Test

Run komiser with the recommended policy and an AWS environment with affected resource.

## Checklist

- [X] Code follows the <a href="https://github.com/tailwarden/komiser/blob/master/CONTRIBUTING.md">contributing</a> guidelines
- [X] Changes have been thoroughly tested
- [ ] <a href="https://github.com/tailwarden/docs.komiser.io">Documentation</a> has been updated, if necessary
- [ ] Any dependencies have been added to the project, if necessary

### CODE REVIEW

File: policy.json [hunk 1]
```diff
@@ -14,6 +19,7 @@
 				"cloudwatch:ListTagsForResource",
 				"cloudwatch:ListDashboards",
 				"cloudwatch:ListMetricStreams",
+				"dynamodb:DescribeTable",
```
Comment (roblambell) [inline policy.json:22]: since https://github.com/tailwarden/komiser/pull/1111

File: policy.json [hunk 2]
```diff
@@ -83,11 +90,12 @@
 				"elasticfilesystem:DescribeFileSystems",
 				"elasticfilesystem:ListTagsForResource",
 				"iam:ListInstanceProfileTags",
+				"lambda:ListEventSourceMappings",
```
Comment (roblambell) [inline policy.json:93]: since https://github.com/tailwarden/komiser/pull/1040

File: policy.json [hunk 3]
```diff
@@ -55,6 +61,7 @@
 				"iam:ListRoles",
 				"iam:ListSAMLProviders",
 				"iam:ListSAMLProviderTags",
+				"iam:ListUsers",
```
Comment (roblambell) [inline policy.json:64]: since https://github.com/tailwarden/komiser/pull/991

File: policy.json [hunk 4]
```diff
@@ -6,6 +6,11 @@
 			"Effect": "Allow",
 			"Action": [
 				"apigateway:GET",
+				"codecommit:ListRepositories",
+				"codecommit:ListTagsForResource",
+				"codebuild:ListProjects",
```
Comment (roblambell) [inline policy.json:11]: since https://github.com/tailwarden/komiser/pull/1216

File: policy.json [hunk 5]
```diff
@@ -6,6 +6,11 @@
 			"Effect": "Allow",
 			"Action": [
 				"apigateway:GET",
+				"codecommit:ListRepositories",
```
Comment (roblambell) [inline policy.json:9]: since https://github.com/tailwarden/komiser/pull/1229

File: policy.json [hunk 6]
```diff
@@ -6,6 +6,11 @@
...
+				"codecommit:ListTagsForResource",
+				"codebuild:ListProjects",
+				"codedeploy:ListApplications",
```
Comment (roblambell) [inline policy.json:12]: since https://github.com/tailwarden/komiser/pull/1228

File: policy.json [hunk 7]
```diff
@@ -83,11 +90,12 @@
 				"elasticfilesystem:DescribeFileSystems",
 				"elasticfilesystem:ListTagsForResource",
 				"iam:ListInstanceProfileTags",
+				"lambda:ListEventSourceMappings",
 				"lambda:ListFunctions",
 				"lambda:ListTags",
 				"es:ListDomainNames",
 				"es:DescribeDomains",
-				"s3:ListBucket"
```
Comment (roblambell) [inline policy.json:90]: This permission is for listing objects/contents of a bucket

> Grants permission to list some or all of the objects in an Amazon S3 bucket (up to 1000).

https://aws.permissions.cloud/iam/s3#s3-ListBucket

File: policy.json [hunk 8]
```diff
@@ -83,11 +90,12 @@
...
+				"s3:ListAllMyBuckets"
```
Comment (roblambell) [inline policy.json:98]: This permission is required for the S3.ListBuckets API method

> list all buckets owned by the authenticated sender of the request
https://aws.permissions.cloud/api/s3#S3_ListBuckets

**Review by Azanul — APPROVED** [review]

**Pull request merged**